ante/ante.go

```diff
@@ -48,11 +47,10 @@ func NewAnteHandler(opts HandlerOptions) (sdk.AnteHandler, error) {
 		ante.NewTxTimeoutHeightDecorator(),
 		ante.NewValidateMemoDecorator(opts.AccountKeeper),
 		ante.NewConsumeGasForTxSizeDecorator(opts.AccountKeeper),
-		NewBypassMinFeeDecorator(opts.BypassMinFeeMsgTypes),
+		NewBypassMinFeeDecorator(opts.BypassMinFeeMsgTypes, opts.GlobalFeeSubspace),
```

Comment:
It's cleaner to instead pass the keeper directly here instead of it's x/param subspace (which is being deprecated anyway).

Reply:
going to leave this for when params is removed since this PR has been open so long and this seems like a "nice to have"